### PR TITLE
CRNS-85 and Login and Registration screens integrated with MOCK data.

### PR DESCRIPTION
**Pull request by harishintimetec**

### CODE REVIEW

File: application/www/js/app.js [hunk 1]
```diff
@@ -17,7 +15,12 @@ angular.module('CRNS', ['ionic', 'CRNSCtrl', 'CRNSSrv'])
   });
 
   $rootScope.$on('$stateChangeSuccess', function(event, toState, toParams, fromState, fromParams) {
-      console.log(toState);
+      //console.log(toState);
+  });
+
+  /* Toaster disappear listner */
+  $rootScope.$on('toasterDisappear', function(e) {
+      Constant.isToaster = true;
```
Comment (inwinderkochhar) [inline application/www/js/app.js]: Constant should be all caps i.e. CONSTANT.IS_TOASTER

File: application/www/js/constants.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+(function() {
+    'use strict';
+    angular.module('CRNSConstants', []).constant('Constant', {
+        'API_URL': 'http://mock.crns/api/',
+        'isToaster': true,
+        'isEmail': /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
```
Comment (inwinderkochhar) [inline application/www/js/constants.js]: Contants should be defined all caps

File: application/www/views/login.html [hunk 3]
```diff
@@ -6,7 +6,7 @@
 
         <div class="list list-inset">
             <label class="item item-input extendInput">
-                <input type="text" placeholder="Username" ng-model="login.username">
+                <input type="text" placeholder="Username" ng-model="login.userName">
```
Comment (inwinderkochhar) [inline application/www/views/login.html]: This is changed again it should be login.username not login.userName

File: application/www/views/registration.html [hunk 4]
```diff
@@ -1,11 +1,11 @@
-<ion-modal-view>
+<ion-view>
     <ion-content has-bouncing="false" class="padding25 alignCenter">
 
         <img src="./img/logo.png">
         <div class="list list-inset alignLeft margin5">
 
             <label class="item item-input extendInput">
-                <input type="text" placeholder="Username" ng-model="register.username">
+                <input type="text" placeholder="Username" ng-model="register.userName">
```
Comment (inwinderkochhar) [inline application/www/views/registration.html]: This is changed again it should be login.username not login.userName

**Comment by harishintimetec** [comment]
@inwinderkochhar  I have changed all constants to caps. Please review it.

**Pull request merged**